src/codeu/chat/server/Model.java

```diff
@@ -125,4 +129,19 @@ public void add(Message message) {
   public StoreAccessor<String, Message> messageByText() {
     return messageByText;
   }
+
+  public void add(Uuid userid, Uuid interest) {
```

Comment:
Wow, that's... not what I would have done. However, your point is well taken: Consistency is more important here, and the function name should stay what it is.